getting-started/FAQ.md

```diff
@@ -0,0 +1,33 @@
+# Activiti Cloud FAQ
+
+## postgreSQL configuration
+our activiti cloud full example rely on postgres for persistent data storage. But when I deploy activiti cloud on kubernetes( using kubeadm, single node cluster) I found our activiti cloud full example stuck. I tried several methods to find out the reason, finally I found it was nothing dealing with activiti cloud.
+The steps to reproduce this situation:
+* helm install --name my-release stable/postgresql
+* open kubernetes dashboard, default namespace.
+* you will find error: unbound immediate persistentvolumeclaims
+
+It is because my kubernetes cluster lack of persistent Volume. 
+Some articles said that when you create persistent volume claims, kubernetes will automatically bind matched pv to your pvc. 
+But my kubernetes cluster is just a single node cluster on CentOS7. It didn't have any cloud stoarge or nfs etc.
+So I have to create my own persistent volume, using this script:
+```
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: local-directory-pv
+  labels:
+    type: local
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+  - ReadWriteOnce
+  hostPath:
+    path: /data/postgres_data
+```
+
+Then open postgres yml config file(using dashboard), remove one line:   "subPath":"postgresql-db", then click update button.
```

Comment:
Thanks for this solution. I cannot find "subPath":"postgresql-db". Which specific "postgres yml config file" is it? Thanks!

Reply:
@ouyangshixiong can you please help us with this question?

Reply:
There is a solution..... The problem is, the AWS, GCP, etc. provide persistent storage which will be bound via the persistent storage claims. Having your non-AWS/GCP, for example using kubeadm/minikube, you have to provide those two persistent volumes 'yourself' ensuring the match the respective activity 7 PV claims originated in rabbitmq and postgres. If this is the case, activiti 7 runs smoothly! Let me know if anybody needs some more information.

Reply:
Sorry for reply so late. Some guy asked me the same question about  "subPath":"postgresql-db" , I didn't face such a problem on my local k8s cluster. After several day ,he told me that: "subPath":"postgresql-db" is not necessary, I think you can just ignore it. @salaboy @vga16